lib/xendit_api/api/virtual_account.rb

```diff
@@ -7,9 +7,10 @@ class VirtualAccount < XenditApi::Api::Base
       PATH = '/callback_virtual_accounts'.freeze
 
       def create(params)
-        params[:expected_amount] = params[:amount] if params[:expected_amount].nil?
         params[:is_closed] = true if params[:is_closed].nil?
         params[:is_single_use] = true if params[:is_single_use].nil?
+        params[:expected_amount] = params[:expected_amount] || params[:amount] unless params[:expected_amount].nil? && params[:amount].nil?
```

Comment:
how about this lek?
```
params[:expected_amount] = params[:amount] unless params[:amount].nil?
```

We only set the expected_amount params explicitly when the params amount is present?